helper/cli.c

```diff
@@ -384,6 +693,18 @@ static struct cli_def *create_cli(cli_shm_t *shm)
 	cli_register_command(cli, c, "odp_pktio_print",
 			     cmd_call_odp_pktio_print,
 			     PRIVILEGE_UNPRIVILEGED, MODE_EXEC, "<name>");
+	cli_register_command(cli, c, "odp_pktio_stats_print",
+			     cmd_call_odp_pktio_stats_print,
+			     PRIVILEGE_UNPRIVILEGED, MODE_EXEC, "<name>");
+	cli_register_command(cli, c, "odp_pktio_queue_stats_print",
+			     cmd_call_odp_pktio_queue_stats_print,
+			     PRIVILEGE_UNPRIVILEGED, MODE_EXEC, "<name>");
+	cli_register_command(cli, c, "odp_pktio_event_queue_stats_print",
+			     cmd_call_odp_pktio_event_queue_stats_print,
+			     PRIVILEGE_UNPRIVILEGED, MODE_EXEC, "<name>");
```

Comment:
Naming is perhaps a bit confusing since there are no such API calls as `odp_pktio_queue_stats_print()` and `odp_pktio_event_queue_stats_print()`. So these commands could be split into input and output specific ones (e.g. `odp_pktin_queue_stats()` and `odp_pktout_queue_stats()`).

Reply:
The intent here is that it is more convenient for CLI user to get both input and output queue dumps with a single command and it's clear enough what the user will get even though the commands don't exactly match an API call.

Reply:
OK, thanks, wasn't aware of the history of the architecture, changed to parentless in v4.